### DISCUSSION TITLE
Can you pleas help me

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#124] [Q&A]
The idea is that if click on animation or swipe up to the animation which I made ,it will change the animation . By just clicking to the animation it will change it. Or by just swiping up . Please help me by pasting the code here. 

I like you guys to paste the code because it really help me to study the code properly.so please paste it here. 

Thanks

**Comment by htoooth** [comment]
@alexmathew11 
<img width="936" alt="image" src="https://user-images.githubusercontent.com/1717023/228713120-4d6e7ec1-23ab-4222-b170-623b1cac02c2.png">

https://docs.zepp.com/zh-cn/docs/1.0/reference/device-app-api/hmUI/widget/IMG_ANIM/?_highlight=ani

You can use the click_down event and then set the animation state in the callback.
